change/@microsoft-teams-js-d1f40ab4-e602-4977-9be1-67eb2a22440e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added SSR support patch",
```

Comment:
Since this will turn into customer facing patch notes, it would be good to be slightly more explicit. Maybe something like "Fixed bugs preventing the use of this library in SSR applications"